src/sdk/network/constants.ts

```diff
@@ -118,6 +138,136 @@ export const Networks: {
       url: '',
     },
   },
+  [1]: {
+    chainId: 1,
+    bundler: 'https://ethereum-bundler.etherspot.io/',
+    contracts: {
+      entryPoint: '0x5FF137D4b0FDCD49DcA30c7CF57E578a026d2789',
+      walletFactory: '0x2f771DCa6Ffa3879e48355E8A4aF5b81d82A6164',
+      uniswapV3SwapRouter: '0xE592427A0AEce92De3Edee1F18E0157C05861564',
```

Comment:
Do we need to store uniswapV3Router here? Can we move it to a separate constants file? Let's retain this only for our contracts?

Reply:
actually we don't need to store the uniswap router since we don't use it anywhere